### PR TITLE
Task: 2205981 Juan Diego Cuevas Mena

### PR DESCRIPTION
**Pull request by code-mena**
.py file added with my name and student code.

### CODE REVIEW

**Comment by NEV117** [comment]
hola, estoy revisando

File: juan_diego_cuevas_2205981.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Nombre completo: Juan Diego Cuevas Mena
```
Comment (NEV117) [inline juan_diego_cuevas_2205981.py:1]: :D

**Review by NEV117 — COMMENTED** [review]
todo good

File: juan_diego_cuevas_2205981.py (hunk 1 above)
Comment (code-mena) [reply juan_diego_cuevas_2205981.py:1]: :D

**Review by NEV117 — APPROVED** [review]

**Pull request merged**